schema/v1.0/parser_test.go

```diff
@@ -170,12 +171,12 @@ func TestFailParseSchema(t *testing.T) {
 
 func TestCheckFileFormatField(t *testing.T) {
 	// Invalid file format version numbers.
-	assert.Error(t, checkFileFormatField("not a semver"))
-	assert.Error(t, checkFileFormatField("2.0.0"))
-	assert.Error(t, checkFileFormatField("1.1.0"))
+	assert.Error(t, internal.CheckFileFormatField("not a semver", supportedFormatMajor, supportedFormatMinor))
+	assert.Error(t, internal.CheckFileFormatField("2.0.0", supportedFormatMajor, supportedFormatMinor))
+	assert.Error(t, internal.CheckFileFormatField("1.1.0", supportedFormatMajor, supportedFormatMinor))
 
 	// Valid cases.
-	assert.NoError(t, checkFileFormatField("1.0.0"))
-	assert.NoError(t, checkFileFormatField("1.0.1"))
-	assert.NoError(t, checkFileFormatField("1.0.10000-alpha+4857"))
+	assert.NoError(t, internal.CheckFileFormatField("1.0.0", supportedFormatMajor, supportedFormatMinor))
+	assert.NoError(t, internal.CheckFileFormatField("1.0.1", supportedFormatMajor, supportedFormatMinor))
+	assert.NoError(t, internal.CheckFileFormatField("1.0.10000-alpha+4857", supportedFormatMajor, supportedFormatMinor))
 }
```

Comment:
Can this be moved to a new `schema/internal/parser_checks_test.go`?

Reply:
Done.